index.html

```diff
@@ -1101,10 +1101,20 @@
 
 Where possible, [=user agents=] should prevent people from being [=cross-context
 recognition|recognized=] across [=partitions=] unless they intend to be
-recognized. Note that sites can do harm even if they can't be completely certain
-that visits come from the same person, so [=user agents=] should also take steps
-to prevent such probabilistic recognition. The [[[Privacy-Threat]]] discusses
-the tradeoffs involved.
+recognized. Note that:
+
+* Sites can do harm even if they can't be completely certain
+  that visits come from the same person, so [=user agents=] should also take steps
+  to prevent such probabilistic recognition. The [[[Privacy-Threat]]] discusses
+  the tradeoffs involved.
+* If a [=user agent=] blocks [=recognition=] between different sites but doesn't
+  notice when a single site contains multiple [=contexts=], this can encourage
+  [=actors=] to bundle multiple contexts into a single site and privilege the
+  [=actors=] who can do so. Despite these harms, even if a [=user agent=] can't
+  always block [=cross-context recognition=] within a single site, it should
+  still prevent [=cross-site recognition=] when it can. By having [=user
```

Comment:
That works for me. I split up your wording to put the context-detection part next to our existing text about detecting sub-site contexts.

I am nervous about over-promising here&mdash;I'm not confident browsers will be able to reliably cut up sites into multiple partitions&mdash;but I don't mind saying that we should try.

Reply:
Thank you -- I agree, it will be harder for browsers to identify separate contexts within a site than contexts that have their own sites (but blocking fingerprinting is hard too, and browsers can do a lot to deter investments in fingerprinting just by declaring that they're going after it where they can)